### PR TITLE
Add screenshots to landing page

### PR DESCRIPTION
**Pull request by perhegg**
#### Description
Changes proposed in this pull request:

* Replaces dummt screenshots with actual screenshots from the site
#### What I have learned working on this feature: 
I have learnt how to modify further semantic ui elements

#### Screenshots:
Skärmklipp

### CODE REVIEW

**Comment by amandagibson** [comment]
can you post a screenshot of the landing page?

**Comment by LauraRe** [comment]
I think the dashboard screenshot would look better in the middle

File: src/components/landingPage.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import React, { Component } from 'react';
 import { Button, Container, Grid, Image, Divider } from 'semantic-ui-react'
-import dummy from '../img/dummy_image.jpg'
+import dashboard from '../img/dashboard.png'
+import graphs from '../img/graphs.png'
+import times from '../img/times.png'
+
+
```
Comment (LauraRe) [inline src/components/landingPage.js]: remove space

File: src/components/landingPage.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 
 class LandingPage extends Component {
```
Comment (LauraRe) [inline src/components/landingPage.js]: remove space

Comment (perhegg) [reply src/components/landingPage.js]: done

File: src/components/landingPage.js (hunk 1 above)
Comment (perhegg) [reply src/components/landingPage.js]: done

**Pull request merged**